pkgs/sdk/server/src/LdClient.cs

```diff
@@ -136,6 +136,12 @@ public LdClient(Configuration config)
             _log = logConfig.LogAdapter.Logger(logConfig.BaseLoggerName ?? LogNames.DefaultBase);
             _log.Info("Starting LaunchDarkly client {0}",
                 AssemblyVersions.GetAssemblyVersionStringForType(typeof(LdClient)));
+
+            if (_configuration.SdkKey == null)
```

Comment:
So, this is where I would expect `if (_configuration.SdkKey == null && !_configuration.offline)`, but it doesn't hurt to make them provide an empty string instead in that case. If we used nullable reference types they probably couldn't pass null.

Reply:
Oh, but they cannot set an empty string with the other validation. I fear this change probably would log errors in existing offline usage.